release.json

```diff
@@ -7,6 +7,14 @@
 		"JMXFETCH_VERSION": "0.27.0",
 		"JMXFETCH_HASH": "52aabf81abb4e42adcb28164b263f01b293dda1c686f845d133ebcd128f8b2ee"
 	},
+	"6.11.3-rc.1": {
+		"INTEGRATIONS_CORE_VERSION": "6.11.0",
+		"PROCESS_AGENT_VERSION": "6.11.1",
+		"OMNIBUS_SOFTWARE_VERSION": "6.11.0",
+		"OMNIBUS_RUBY_VERSION": "6.11.x",
+		"JMXFETCH_VERSION": "0.27.0",
```

Comment:
we should update the JMXFETCH_VERSION as well to include this change https://github.com/DataDog/jmxfetch/pull/229

Reply:
this should point to `0.27.1`, hashsum `dfc027a1f38558a06b9ce3a86c50b89875aba2455110c8614e93405a5b6ff2c2` (you can check against https://bintray.com/datadog/datadog-maven/jmxfetch)